<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="zMachine" failures="5" tests="8" name="BankingApp Test" time="75.224" errors="0" timestamp="20 Feb 2020 02:41:08 GMT">
  <testcase classname="com.inetBanking.testCases.TC_LoginTest_001" name="loginTest" time="1.602"/>
  <testcase classname="com.inetBanking.testCases.TC_LoginTestDDT_002" name="loginDDT" time="7.266"/>
  <testcase classname="com.inetBanking.testCases.TC_LoginTestDDT_002" name="loginDDT" time="2.054">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCases.TC_LoginTestDDT_002.loginDDT(TC_LoginTestDDT_002.java:31)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.inetBanking.testCases.TC_LoginTestDDT_002" name="loginDDT" time="2.451">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCases.TC_LoginTestDDT_002.loginDDT(TC_LoginTestDDT_002.java:31)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.inetBanking.testCases.TC_LoginTestDDT_002" name="loginDDT" time="1.967">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCases.TC_LoginTestDDT_002.loginDDT(TC_LoginTestDDT_002.java:31)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.inetBanking.testCases.TC_LoginTestDDT_002" name="loginDDT" time="2.476">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCases.TC_LoginTestDDT_002.loginDDT(TC_LoginTestDDT_002.java:31)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.inetBanking.testCases.TC_LoginTestDDT_002" name="loginDDT" time="2.186">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCases.TC_LoginTestDDT_002.loginDDT(TC_LoginTestDDT_002.java:31)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.inetBanking.testCases.TC_AddCustomerTest_003" name="addNewCustomer" time="28.214"/>
</testsuite> <!-- BankingApp Test -->
